python3-sys/src/import.rs

```diff
@@ -83,4 +91,7 @@ pub struct _frozen {
 #[cfg_attr(windows, link(name="pythonXY"))]
 extern "C" {
     pub static mut PyImport_FrozenModules: *const _frozen;
+    pub static mut PyImport_Inittab: *mut _inittab;
```

Comment:
If `_inittab` is conditioned by `Py_LIMITED_API` shouldn't this also?

Reply:
This whole block is protected by a `#[cfg(not(Py_LIMITED_API))]`.